### PR TITLE
Add the Rules module which pulls from a list of rules defined in the module

### PR DESCRIPTION
**Pull request by scmcgowen**

### CODE REVIEW

File: src/modules/Rules.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      data: new SlashCommandBuilder()
+        .setName("rule")
+        .setDescription("Replies with the selected rule")
+        .addIntegerOption((option) =>
+          option
+            .setName("rule")
+            .setDescription("Select a rule")
+            .setRequired(true)
+            .setMinValue(1)
+            .setMaxValue(7)
```
Comment (Lemmmy) [inline src/modules/Rules.ts]: Can you initialize this with `rules.length`?

```suggestion
            .setMaxValue(rules.length)
```

File: src/modules/Rules.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  "3. No NSFW or obscene content. This includes text, images, or links featuring nudity, sex, hard violence, or other graphically disturbing content.",
+  "4. If you see something that makes you feel unsafe, let staff know by ping or direct message. We want this server to be a welcoming space!",
+  "5. Your (server-)username should be something that is both pronounceable and typeable using an (US)-International keyboard without modifier keys.",
+  "6. Have a good time and let us know how we can improve!",
+  "7. Do not feed trolls or fuel bullying, harassment or other unwanted behaviour. Stop participating in the conversation and report to staff.",
+];
+
+export default class Rules extends Module {
+  name = "Ping";
+  description = "This module allows users to ping the bot and receive a reply.";
```
Comment (Lemmmy) [inline src/modules/Rules.ts]: Don't forget to update these

**Review by Lemmmy — CHANGES_REQUESTED** [review]

File: src/modules/Rules.ts (hunk 2 above)
Comment (scmcgowen) [reply src/modules/Rules.ts]: Will fix. Didn't notice that

**Comment by scmcgowen** [comment]
Fixed those issues

**Comment by Lemmmy** [comment]
Thanks for the PR!

**Pull request merged**